Example/Messaging/Tests/FIRMessagingAnalyticsTest.m

```diff
@@ -407,13 +407,15 @@ - (void)testNoConversionTracking {
   [FIRMessagingAnalytics logUserPropertyForConversionTracking:notification toAnalytics:analytics];
 }
 
+#if TARGET_OS_IOS || TARGET_OS_TV
```

Comment:
Shouldn't this only be TARGET_OS_IOS since there is no Analytics for tvOS?

Reply:
Good point.